MiAZ/miaz.py

```
@@ -1,111 +1,59 @@
 from MiAZ.backend.util import which
 
-VERSION = '0.1.0'
 log = MiAZLog('MiAZ')
-ENV = {}
 
-# Desktop environment
+# Check Desktop environment
 ENV['DESKTOP'] = {}
 try:
     import gi
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)